runner/app/pipelines/image_to_video.py

```diff
@@ -113,6 +136,14 @@ def __call__(
         seed = kwargs.pop("seed", None)
         safety_check = kwargs.pop("safety_check", True)
 
+        if self.pipeline_name == "LTXImageToVideoPipeline":
```

Comment:
Can we do something more generic that looks at the pipeline class args and del the kwargs keys if not present in the pipeline? If a heavy lift we can do in separate PR.

Reply:
Good point would love to make all pipelines more generic. We can do this in seperate PR.